### PR TITLE
First schedule can have any #from date

### PR DESCRIPTION
**Pull request by AndrewVos**
~~This is WIP at the moment because I'm not sure if the UI will work properly with
the code changes.~~

Feel free to review though!

### CODE REVIEW

File: app/models/schedule.rb [hunk 1]
```diff
@@ -5,10 +5,17 @@ class Schedule < ApplicationRecord
...
   validates :from, presence: true
-  validate :from_must_be_more_than_six_weeks_in_the_future
+  validate :from_must_be_more_than_six_weeks_in_the_future, unless: :first_schedule?
 
   private
 
+  def first_schedule?
+    user
+      .schedules
+      .select(&:persisted?)
```
Comment (benlovell) [inline app/models/schedule.rb]: You could replace the `select` `empty?` with `none?(&:persisted?)`.

File: spec/models/schedule_spec.rb [hunk 2]
```diff
@@ -3,17 +3,48 @@
 RSpec.describe Schedule, type: :model do
   describe 'validation' do
     context '#from' do
-      it 'is valid with valid attributes' do
-        expect(build(:schedule, from: 6.weeks.from_now.beginning_of_day)).to be_valid
-      end
+      context 'first schedule' do
+        let :user do
```
Comment (benlovell) [inline spec/models/schedule_spec.rb]: Nit-pick but can we wrap the `let` arguments in parens? This seems to be the prevalant style.

File: app/models/schedule.rb (hunk 1 above)
Comment (AndrewVos) [inline app/models/schedule.rb]: :+1:

File: spec/models/schedule_spec.rb (hunk 2 above)
Comment (AndrewVos) [inline spec/models/schedule_spec.rb]: done

**Pull request merged**